pkg/generate/app/sourcelookup.go

```diff
@@ -90,6 +90,7 @@ type SourceRepository struct {
 	buildWithDocker  bool
 	ignoreRepository bool
 	binary           bool
+	dockerfile       bool
```

Comment:
comment this field or rename it.

It means that the repository has an explicitly provided dockerfile from the CLI, not to be confused with meaning the git repo contains a dockerfile.